.github/scripts/generate_ci_workflows.py

```diff
@@ -228,6 +233,12 @@ def generate_workflow_file(self, workflow_template: jinja2.Template) -> None:
         enable_nogpu_no_avx2_test=1,
         enable_slow_test=1,
         num_test_shards=2,
+        on_pull_request=True,
+        ciflow_config=CIFlowConfig(
+            enabled=True,
+            trigger_action_only=True,
+            labels=set(['ciflow/slow']),
```

Comment:
nit:
```suggestion
            labels={'ciflow/slow'},
```

Reply:
ha, always forget the set literal, will change in the following PRs